website/blog/2023-03-21-5-things-to-know-for-a-docker-user.md

```diff
@@ -0,0 +1,109 @@
+description: Important thinks to know when switching from Docker Desktop to Podman Desktop
+slug: 5-things-to-know-for-a-docker-user
+authors: [benoitf]
+tags: [podman-desktop, docker, migrating]
+hide_table_of_contents: false
+---
+
+The 5 things to know being a Docker user by using Podman Desktop:
+
+- **Use a single UI**: Podman Desktop works with several container engines, including Docker.
+- The **compatibility mode**: How to ensure tools are working with Podman instead of Docker.
+- **Compose** Support. How to work with Compose files and Podman.
+- **Kubernetes**. How to use Kubernetes with Podman.
+- **rootless** mode or containers without root privileges.
+
+
+![5-things-to-know-for-a-docker-user-hero](img/5-things-to-know-for-a-docker-user/5-things-to-know-for-a-docker-user-hero.png)
+
+<!--truncate-->
+_________________
+
+## Use Podman Desktop to interact with containers running in Docker
+
+Docker Desktop provides a UI to interact with containers and images. But the UI depends on Docker API and it is not possible to use the UI with Docker and Podman at the same time.
+
+Podman Desktop is a multi-engine UI tool. The UI is compatible with the API of Docker and Podman. It means all containers and images from all the engines at the same time are visible in the UI.
+
+When migrating from Docker to Podman, you can use Podman Desktop to interact with containers running in Docker. Explore all commands and features of Podman Desktop and see all the resources from Docker.
+
+![Many container engines at the same time](img/5-things-to-know-for-a-docker-user/multiple-container-engines.png)
+
+## Docker compatibility mode
+
+Using Podman with Podman Desktop or with the CLI is straightforward. But some tools expect to find `docker` CLI or `docker.sock` socket. In this case, you have to use the compatibility mode of Podman.
+
+### Socket file compatibility
+
+The socket compatibility mode is a feature of Podman that allows to bind the Podman socket under the Docker socket path.
+
+On Windows it is always enabled by default. On macOS, by using the `.pkg installer` it is active by default but not by using the `brew` installation because it requires some admin permissions.
+
+You can enable it by [invoking a CLI tool](https://podman-desktop.io/docs/migrating-from-docker/using-podman-mac-helper)
+
+For example if you use [`TestContainers`](https://www.testcontainers.org/) in your Java project, you can use the compatibility mode to ensure that the tool will use Podman instead of Docker.
+
+### CLI compatibility
+
+If you have scripts relying on `docker` CLI, you can use the compatibility mode to ensure that the tool is working with Podman instead of Docker.
+
+[Create a script](https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman) called `docker` that will call the `podman` CLI
+
+**_NOTE:_** creating a shell prompt alias, for example `alias docker=podman`, will not work inside scripts that you call.
+
```

Comment:
I think we should detail the 2 following cases:

- Docker installed on the machine
- Docker not installed on the machine

Reply:
added the 2 cases